packages/react-components/theme-designer/src/components/ColorTokens/ColorTokens.tsx

```diff
@@ -52,8 +47,9 @@ const darkBrandColors: ColorOverrideBrands = getOverridableTokenBrandColors(team
 export const ColorTokens: React.FunctionComponent<ColorTokensProps> = props => {
   const styles = useStyles();
 
-  const { brand, isDark, theme, themeLabel, dispatchAppState } = props;
-
+  const { theme } = props;
+  const { appState, dispatchAppState } = React.useContext(AppStateContext);
```

Comment:
We have a `useContextSelector` hook exported from `@fluentui/react-context-selector` that has some performance improvements over `React.useContext`. I think it might be good to use here, like so:

```ts
const appState = useContextSelector(AppStateContext, ctx => ctx.appState);
const dispatchAppState = useContextSelector(AppStateContext, ctx => ctx.dispatchAppState);
```